app/api/v2/identity/sessions.rb

```diff
@@ -87,6 +96,67 @@ class Sessions < Grape::API
           session.destroy
           status(200)
         end
+
+        namespace :auth0 do
+          desc 'Auth0 authentication',
+               success: { code: 200, message: '' },
+               failure: [
+                 { code: 400, message: 'Required params are empty' },
+                 { code: 404, message: 'Record is not found' }
+               ]
+
+          get '/authorize' do
+            uri = URI::HTTPS.build(host: Barong::App.config.auth0_tenant_address, path: '/authorize', query: {
+              response_type: 'code',
+              client_id: Barong::App.config.client_id,
+              redirect_uri: Barong::App.config.redirect_uri,
+              scope: 'openid profile email',
```

Comment:
which scopes we can put here?
If its kind of customization, maybe we should do this with env too